src/storage/storage.cc

```diff
@@ -337,15 +344,20 @@ Status Storage::Open(DBOpenMode mode) {
   propagate_opts.disable_auto_compactions = config_->rocks_db.disable_auto_compactions;
   SetBlobDB(&propagate_opts);
 
+  rocksdb::ColumnFamilyOptions minor_opts(subkey_opts);
+  minor_opts.write_buffer_size = config_->minor_columns_write_buffer_size * KiB;
+  pubsub_opts.write_buffer_size = config_->minor_columns_write_buffer_size * KiB;
+  propagate_opts.write_buffer_size = config_->minor_columns_write_buffer_size * KiB;
```

Comment:
Could you explain for these two lines of code?

Reply:
Don't we want to specify column families except for metadata and subkey?

Reply:
What will happen if `minor_columns_write_buffer_size` is disabled?